idl/gimlet-hf.idol

```diff
@@ -57,5 +57,28 @@ Interface(
                 err: CLike("HfError"),
             ),
         ),
+	"get_mux": (
```

Comment:
Why are these de-dented like this?

Reply:
Ha ha -- it's an errant hard tab because my editor didn't recognize `.idl` when I first started editing them.  Fixed; thank you!